airflow/models/pool.py

```diff
@@ -162,19 +163,19 @@ def slots_stats(
 
         pools: dict[str, PoolStats] = {}
 
-        query = session.query(Pool.pool, Pool.slots)
+        query = select(Pool.pool, Pool.slots)
 
         if lock_rows:
             query = with_row_locks(query, session=session, **nowait(session))
 
-        pool_rows: Iterable[tuple[str, int]] = query.all()
+        pool_rows: Iterable[Row] = session.execute(query).all()
```

Comment:
So unfortunate this is actually worse than the `tuple` typing since we can no longer express the innter types 😞

Reply:
I have removed the typing. I think there's no need for it